pennylane_lightning/core/src/simulators/lightning_kokkos/StateVectorKokkos.cpp

```diff
@@ -19,5 +19,5 @@
 
 #include "StateVectorKokkos.hpp"
 
-template class Pennylane::LightningKokkos::StateVectorKokkos<float>;
-template class Pennylane::LightningKokkos::StateVectorKokkos<double>;
+extern template class Pennylane::LightningKokkos::StateVectorKokkos<float>;
+extern template class Pennylane::LightningKokkos::StateVectorKokkos<double>;
```

Comment:
This likely shouldn't be extern --- it is telling the library that the objects are built elsewhere, but this is the file that we want to use to compile them for the .so